xbmc/music/MusicDatabase.cpp

```diff
@@ -1923,6 +1938,35 @@ bool CMusicDatabase::GetArtistsBySong(int idSong, std::vector<int> &artists)
   return false;
 }
 
+bool CMusicDatabase::GetSongByDiscSubtitleAndAlbum(const std::string& strDiscSubtitle, int idAlbum)
+{
```

Comment:
A redundant function

Reply:
Yeah I know.  It was added originally when I wasn't sure exactly what routines I would need.